Add dns attribute to the naming convention (#6179)
DataDog/documentation: content/en/logs/processing/attributes_naming_convention.md

Some integrations that rely on these are: [Rsyslog][15], [NxLog][16], [Syslog-ng][17], [Fluentd][18], [Logstash][19], etc.

### DNS

All attributes and measures are prefixed by `dns`.

| **Fullname**         | **Type** | **Description**                                                           |
|:---------------------|:---------|:--------------------------------------------------------------------------|
| `dns.id`             | `string` | The DNS query identifier.                                                 |

> | `dns.question.name`  | `string` | The IP address URL the DNS question wish to find.                         |

Reviewer: ```suggestion
| `dns.question.name`  | `string` | The IP address URL that the DNS question wishes to find.                         |
```

> | `dns.question.type`  | `string` | A two octet code which specifies the DNS question type (example "AAAA").  |

Reviewer: ```suggestion
| `dns.question.type`  | `string` | A two octet code which specifies the DNS question type (example: "AAAA").  |
```

> | `dns.question.type`  | `string` | A two octet code which specifies the DNS question type (example: "AAAA").  |

Reviewer 2: The 2 octet code is what is present in the DNS packet but here it is in human readable form (same for the answer type)
As a reference: https://en.wikipedia.org/wiki/List_of_DNS_record_types

Reviewer 3: added ref link

| `dns.question.class` | `string` | The class looked up by the DNS question (i.e IN when using the internet). |
| `dns.question.size`  | `number` | The DNS question size in bytes.                                           |
| `dns.answer.name`    | `string` | The queried domain name.                                                  |

> | `dns.answer.type`    | `string` | A two octet code which specifies the DNS answer type .                    |

Reviewer: ```suggestion
| `dns.answer.type`    | `string` | A two octet code which specifies the DNS answer type.                    |
```
